plugin.video.hbogoeu/hbogolib/base.py

```diff
@@ -0,0 +1,192 @@
+# encoding: utf-8
+# base class for Hbo Go Kodi add-on
+# Copyright (C) 2019 ArvVoid (https://github.com/arvvoid)
+# Relesed under GPL version 2
+#########################################################
+
+from __future__ import absolute_import, division
+
+import sys
+import traceback
+
+from hbogolib.constants import HbogoConstants
+
+try:
+    import urlparse as parse
+    from urllib import unquote_plus as unquote
+except ImportError:
+    import urllib.parse as parse
+    from urllib.parse import unquote_plus as unquote
+
+from kodi_six import xbmc, xbmcaddon, xbmcgui
+
+
+class hbogo(object):
+
+    def __init__(self, handle, base_url):
+        self.base_url = base_url
+        self.handle = handle
+        self.addon = xbmcaddon.Addon()
+        self.addon_id = self.addon.getAddonInfo('id')
+        self.language = self.addon.getLocalizedString
+        self.handler = None
+
+    @staticmethod
+    def country_index(country_id):
+        index = -1
+
+        for i in range(len(HbogoConstants.countries)):
+            if HbogoConstants.countries[i][1] == country_id:
+                index = i
+                break
+
+        return index
+
+    def start(self):
+        country_id = self.addon.getSetting('country_code')
+        country_index = self.country_index(country_id)
+
+        if country_index == -1:
+            self.setup()
+            country_id = self.addon.getSetting('country_code')
+            country_index = self.country_index(country_id)
+            if country_index == -1:
+                xbmcgui.Dialog().ok("ERROR", "Setup failed")
```

Comment:
please localize this strings, they are presented to the user on the interface

Reply:
Fixed tnx